dev/sg/internal/rfc/rfc.go

```diff
@@ -0,0 +1,189 @@
+	homePath, err := root.GetSGHomePath()
+	if err != nil {
+		return nil, err
+	}
+	fp := filepath.Join(homePath, googleTokenFileName)
+
+	// Try to read token from file...
+	tok, err := readTokenFromFile(fp)
+	if err != nil {
+		// ...if it doesn't exist, open browser and ask user to give us
+		// permissions
+		tok, err = getTokenFromWeb(ctx, config, out)
+		if err != nil {
+			return nil, err
+		}
+		if err := saveToken(fp, tok); err != nil {
+			return nil, err
+		}
+	}
+
+	return config.Client(ctx, tok), nil
+}
+
+// Request a token from the web, then returns the retrieved token.
+func getTokenFromWeb(ctx context.Context, config *oauth2.Config, out *output.Output) (*oauth2.Token, error) {
+	authURL := config.AuthCodeURL("state-token", oauth2.AccessTypeOffline)
+
+	out.Writef("Opening %s ...", authURL)
+	if err := urlutil.OpenURL(authURL); err != nil {
+		return nil, err
+	}
+
+	fmt.Printf("Paste the OAuth token here: ")
+	var authCode string
+	if _, err := fmt.Scan(&authCode); err != nil {
+		return nil, err
+	}
+
+	return config.Exchange(ctx, authCode)
+}
+
+// Retrieves a token from a local file.
+func readTokenFromFile(file string) (*oauth2.Token, error) {
+	f, err := os.Open(file)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	tok := &oauth2.Token{}
+	return tok, json.NewDecoder(f).Decode(tok)
+}
+
+// Saves a token to a file path.
+func saveToken(path string, token *oauth2.Token) error {
+	fmt.Printf("Saving credential file to: %s\n", path)
+	f, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
+	if err != nil {
+		return errors.Wrap(err, "unable to cache oauth token")
+	}
+	defer f.Close()
+
+	return json.NewEncoder(f).Encode(token)
+}
+
+func queryRFCs(ctx context.Context, query string, orderBy string, pager func(r *drive.FileList) error, out *output.Output) error {
+	// If modifying these scopes, delete your previously saved token.json.
+	config, err := google.ConfigFromJSON([]byte(credentials), drive.DriveMetadataReadonlyScope)
+	if err != nil {
+		return errors.Wrap(err, "Unable to parse client secret file to config")
+	}
+	client, err := getClient(ctx, config, out)
+	if err != nil {
+		return errors.Wrap(err, "Unable to build client")
+	}
+
+	srv, err := drive.NewService(ctx, option.WithHTTPClient(client))
+	if err != nil {
+		return errors.Wrap(err, "Unable to retrieve Drive client")
+	}
+
+	if query == "" {
+		query = "name contains 'RFC'"
+	}
+	q := fmt.Sprintf("%s and parents in '1zP3FxdDlcSQGC1qvM9lHZRaHH4I9Jwwa' or %s and parents in '1KCq4tMLnVlC0a1rwGuU5OSCw6mdDxLuv'", query, query)
+
+	list := srv.Files.List().
+		Corpora("drive").SupportsAllDrives(true).
+		DriveId("0AK4DcztHds_pUk9PVA").
+		IncludeItemsFromAllDrives(true).
+		SupportsAllDrives(true).
+		PageSize(100).
+		Q(q).
+		Fields("nextPageToken, files(id, name, parents)")
+
+	if orderBy != "" {
+		list = list.OrderBy(orderBy)
+	}
+
+	return list.Pages(ctx, pager)
+}
+
+func ListRFCs(ctx context.Context, out *output.Output) error {
```

Comment:
`rfc.List` is better than `rfc.ListRFCs`, I think :) That also applies to the other two suggestions further down.

```suggestion
func List(ctx context.Context, out *output.Output) error {
```